src/__tests__/e2e/3.2.0/valid-swaggerSpec/serverless.yml

```diff
@@ -0,0 +1,72 @@
+service: serverless-wso2-apim
```

Comment:
Dude I love this integration testing. What I'd do is add a comment on top of each of these files to as what exactly this test entails to. Kinda like `describe`

Reply:
Thanks mate! Thats the name of the folder (i presumed would be detailed enough to describe test case). So the content of serverless.yml can be as blant as it can, for simplicity. What do you think?

Reply:
Sure that works dude, just thinking if you got a more complex scenario, i'd be hard to capture that in the folder name. But if that's how you set it up then roll with it!